merino-web/src/providers.rs

```diff
@@ -84,6 +84,12 @@ async fn make_provider_tree(
         SuggestionProviderConfig::Fixed(fixed_config) => {
             FixedProvider::new_boxed(settings, fixed_config)?
         }
+
+        SuggestionProviderConfig::KeywordFilter(filter_config) => {
+            let inner = make_provider_tree(settings, filter_config.inner.as_ref()).await?;
+            KeywordFilterProvider::new_boxed(filter_config.suggestion_blocklist.clone(), inner)?
```

Comment:
We could give `KeywordFilterProvider` a reference to a metrics client here, maybe.